engine/engine_test.go

```diff
@@ -555,6 +557,29 @@ func TestBaseOutput(t *testing.T) {
 	assert.Equal(t, "bar", output["foo"])
 }
 
+func TestScriptPlugin(t *testing.T) {
+	file := os.Getenv("SCRIPTS") + "/hello-world.py"
```

Comment:
The test should probably use /bin/sh instead of python to be runnable everywhere